pipeline/pipelines/gapic_generation_pipeline.py

```diff
@@ -52,20 +53,33 @@ def __init__(self, **kwargs):
             **kwargs)
 
 
+class GapicClientBatchPipeline(batch_gen.BatchPipeline):
+
+    def __init__(self, **kwargs):
+        super(GapicClientBatchPipeline, self).__init__(
+            _get_batch_pipeline_tasks, **kwargs)
+
+
+def _get_batch_pipeline_tasks(**kwargs):
```

Comment:
This isn't really actionable, but I wanted to call out that this function is actually a factory factory.